### ISSUE TITLE
accessiblity link tags on the FCC homepage

### ISSUE THREAD

**Issue by toChaim** [freeCodeCamp/freeCodeCamp#14243]
#### Issue Description

Screen readers for blind users automatically read what is inside <a> tags as webpages they could get to. Three of the links on our homepage are not informative about where they lead. They are on the **NUMBER** of code challenges solved, the **NUMBER** of dollars of developer work donated and the **NUMBER**  of people with developer jobs.
To make the webpage more usable for people who are blind, the links should be on the whole statement or the words not an uninformative number. 

#### Screenshot

![freecodecamp](https://cloud.githubusercontent.com/assets/15223068/24682862/bd4439ae-1950-11e7-86d2-94c39b94ea6d.jpg)

**Comment by ganes1410** [comment]
can i do this ??

**Comment by HKuz** [comment]
@toChaim - thanks for noticing this. Your suggestion is timely, considering there's a new challenge on the Beta site about using descriptive link text for accessibility 😄 @QuincyLarson and @BerkeleyTrue - would you guys be okay if we were to change this page so the entire caption is the link text, instead of just the number/dollar amounts?

**Comment by toChaim** [comment]
That would solve the problem. I think it looks cool the way it is. Could we keep the formatting the way it? Or we could move the anchor tag to the description and leave the number out out of it, which look similar.
Am I write that "server\views\home.jade" is the file that needs to be changed?
Anyway thanks for your help
Chaim Finkelman

**Comment by toChaim** [comment]
What is the next step?

**Comment by QuincyLarson** [comment]
@toChaim Thanks for pointing out this accessibility issue. I actually think we should remove those links entirely and just turn them into normal text. I'm working on overhauling the landing page a bit and will do so when I get a chance.
